compositions/CollectionMenu/CollectionMenu.css.ts

```diff
@@ -24,5 +24,52 @@ export const collectionTrigger = style([
 
 export const modalWrapper = style({
   overflowY: 'scroll',
+  msOverflowStyle: 'none',
+  scrollbarWidth: 'none',
+  '::-webkit-scrollbar': {
+    display: 'none',
+  },
   height: 400,
 })
+
+export const tabsButton = style([
+  {
+    all: 'unset',
```

Comment:
Because of the unset, atomizing here might not work :)

Reply:
I removed the unset and styling came out better anyway. Atomized a bunch of it :)